omnibus/config/templates/datadog-agent/systemd.process.service.erb

```diff
@@ -1,16 +1,15 @@
 [Unit]
-Description="Datadog Agent"
-Requisite=datadog-agent.service
-After=network.target datadog-agent.target
-PartOf=datadog-agent.service
+Description="Datadog Process Agent"
+After=network.target datadog-agent.service
+BindsTo=datadog-agent.service
+StartLimitIntervalSec=10
+StartLimitBurst=5
 
 [Service]
```

Comment:
It's maybe usefull to set a `RestartSec=` option

Reply:
we could, the default of `100ms` sounds okay to me though